base/test/routes/jwt_test.mustache

```diff
@@ -24,14 +24,14 @@ describe('JWT authentication handling', () => {
   });
 
   it('must accept a valid JWT in a query string param', () => {
-    return fetch(url+'?jwt='+token, {
+    return fetch(url+'?token='+token, {
```

Comment:
Why is this required? Is it so that the express token parsing middleware picks it up correctly?